middleware/src/main/java/com/frontegg/sdk/middleware/routes/impl/FronteggConfigRoutsService.java

```diff
@@ -0,0 +1,89 @@
+package com.frontegg.sdk.middleware.routes.impl;
+
+import com.frontegg.sdk.api.client.IApiClient;
+import com.frontegg.sdk.common.util.HttpUtil;
+import com.frontegg.sdk.common.util.StringHelper;
+import com.frontegg.sdk.config.FronteggConfig;
+import com.frontegg.sdk.middleware.context.FronteggContextHolder;
+import com.frontegg.sdk.middleware.routes.IFronteggRouteService;
+import com.frontegg.sdk.middleware.routes.model.KeyValPair;
+import com.frontegg.sdk.middleware.routes.model.RoutesConfig;
+import com.frontegg.sdk.middleware.routes.model.VendorClientPublicRouts;
+
+import javax.servlet.http.HttpServletRequest;
+import java.util.List;
+import java.util.Map;
+
+public class FronteggConfigRoutsService implements IFronteggRouteService {
+
+    private IApiClient apiClient;
+    private FronteggConfig fronteggConfig;
+    private RoutesConfig routesConfig;
+
+    private static final String ROUT_PATH = "/configs/routes";
+
+    public FronteggConfigRoutsService(IApiClient apiClient, FronteggConfig fronteggConfig) {
+        this.apiClient = apiClient;
+        this.fronteggConfig = fronteggConfig;
+    }
+
+    private void init() {
+        routesConfig = getRoutesConfig();
+    }
+
+    public boolean isFronteggPublicRoute(HttpServletRequest request) {
+        init();
+
+        if (routesConfig == null || routesConfig.getVendorClientPublicRoutes() == null) return false;
+
+        String path = HttpUtil.getRequestUrl(
+                request.getRequestURI(), FronteggContextHolder.getContext().getFronteggBasePath())
+                .replaceFirst("/", ""
```

Comment:
what will happen if the url doesn't have a "/" prefix? I think it will replace the slash somewhere in the middle of the route